pkg/virt-api/webhooks/validating-webhook/admitters/network.go

```diff
@@ -58,3 +58,24 @@ func validateInterfaceStateValue(field *k8sfield.Path, spec *v1.VirtualMachineIn
 	}
 	return causes
 }
+
+func validateInterfaceBinding(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec) []metav1.StatusCause {
```

Comment:
nit: How about `validateInterfaceBindingPlugin`?

Reply:
It will be incorrect. It checks all binding configuration, even the old one.